File: src/main/java/me/yoursole/ctf/commands/Stop.java
package me.yoursole.ctf.commands;

import me.yoursole.ctf.data.GameData;
import me.yoursole.ctf.data.items.Flag;
import org.bukkit.Bukkit;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;


public class Stop implements CommandExecutor {
    @Override
    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
        if (sender.isOp()) {
            for (Player player : Bukkit.getOnlinePlayers()) {
                player.getInventory().remove(Flag.flag);
            }
            GameData.it = null;
            sender.sendMessage("Game Stopped");
            return true;
        } else {
            sender.sendMessage("don't u dare ._.");
            return true;
        }

    }

}

File: src/main/kotlin/me/yoursole/ctf/commands/Stop.kt
package me.yoursole.ctf.commands

import me.yoursole.ctf.data.GameData
import me.yoursole.ctf.data.items.Flag
import org.bukkit.Bukkit
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender

class Stop : CommandExecutor {
    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        return if (sender.isOp) {
            for (player in Bukkit.getOnlinePlayers()) {
                player.inventory.remove(Flag.flag)
            }
            GameData.it = null
            sender.sendMessage("Game Stopped")
            true
        } else {
            sender.sendMessage("don't u dare ._.")
            true
        }
    }
}